include/crpropa/magneticField/turbulentField/SimpleGridTurbulence.h

```diff
@@ -0,0 +1,115 @@
+#ifndef CRPROPA_SIMPLEGRIDTURBULENCE_H
+#define CRPROPA_SIMPLEGRIDTURBULENCE_H
+
+#ifdef CRPROPA_HAVE_FFTW3F
+
+#include "crpropa/magneticField/turbulentField/GridTurbulence.h"
+
+#include "kiss/logger.h"
+#include "kiss/string.h"
+
+namespace crpropa {
+/**
+ * \addtogroup MagneticFields
+ * @{
+ */
+
+/**
+ @class SimpleTurbulenceSpectrum
+ @brief Defines the energy spectrum of simple power-law turbulence
+ */
+class SimpleTurbulenceSpectrum : public TurbulenceSpectrum {
+  public:
+	/**
+	 @param Brms         root mean square field strength for generated field
+	 @param lMin	 Minimum physical scale of the turbulence
+	 @param lMax	 Maximum physical scale of the turbulence
+	 @param lBendover	   the bend-over scale
```

Comment:
lBendover is not available in SimpleTurbulenceSpectrum, is it?

Reply:
True. What I named the simple turbulence spectrum is just the inertial part of turbulence, i.e. a power-law with cut-offs at k_min and k_max.